### PR TITLE
Fix TransactionOutcome Borsh deserialization by light client

### PR DESCRIPTION
**Pull request by k06a**
This field is truly hard to parse on the light client side. After talk with @nearmax we decided to move this field to the end of each parent struct to allow skip its parsing. Because Borsh doesn't allows us to skip it.

### CODE REVIEW

**Comment by k06a** [comment]
@nearmax should I also update `ExecutionOutcomeView`, `FinalExecutionOutcomeView` and traits like `Debug` etc?

**Comment by nearmax** [comment]
> @nearmax should I also update ExecutionOutcomeView, PartialExecutionOutcome, FinalExecutionOutcomeView

Yes. Since we pass it as JSON to nearlib this should not be a breaking change for the Applayer. 
It will however break the state and will require a hardfork.

> and traits like `Debug` etc?

I am not following, what do you mean?

**Comment by k06a** [comment]
@nearmax I was talking about this: https://github.com/nearprotocol/nearcore/pull/1934/files#diff-3c49701f2550a059dc2b7db73676cd6bR288

**Comment by k06a** [comment]
Rebased one more time, let's merge it

**Comment by nearmax** [comment]
@fckt Could you please take a look whether this is outdated?

**Comment by fckt** [comment]
I haven'n found a project where we use Borsh-serialized `TransactionOutcome`. If there is any, make sure that schema is outdated, since the order of fields in `TransactionOutcome` is changes by this PR @vgrichina @frol
